app/views/shipments/_form.html.erb

```diff
@@ -33,21 +33,20 @@
         title: 'Term',
         name: 'term_code',
         placeholder: "Please select appendix value",
-        data: ['a', 'b', 'c'],
+        data: Term.pluck(:code),
```

Comment:
can we apply explicit alphabetic ordering for all trade codes and geo entities drop downs?